js/interface/map-panel.js

```diff
@@ -60,7 +134,29 @@ Fliplet.InteractiveMap.component('map-panel', {
         }
       })
 
-      window.filePickerProvider.then((result) => {
+      window.filePickerProvider.then(result => {
+       if (this.shouldKeepMarkers) {
```

Comment:
```
if (!this.shouldKeepMarkers) {
  return;
}

// rest of code
```

Reply:
Can't do it because part of the code needs to be executed anyway